libs/application/templates/financial-aid/src/forms/Application.ts

```diff
@@ -96,6 +96,29 @@ export const Application: Form = buildForm({
             }),
           ],
         }),
+        buildSubSection({
+          condition: (answers) => answers.income === ApproveOptions.Yes,
+          id: 'incomeFiles',
+          title: m.incomeFilesForm.general.sectionTitle,
+          children: [
+            buildCustomField({
+              id: routes.INCOMEFILES,
```

Comment:
Just wondering why do most of the custom fields have their id assigned from the routes constant but some don't?

Reply:
We are routing from the summary file also, so to avoid having to change the routes in two places, we put it in one and use it in the file and in summaryform file 😸